conduct.md

```diff
@@ -100,32 +100,46 @@ TODO: Write an introductory paragraph.
 
 ### Conflicts
 
-TODO: Write an introductory paragraph.
+Students experiencing conflicts should come together to fix the problem. No argument
```

Comment:
You might want to split the second sentence into 2 sentences (from where the comma is).